### DISCUSSION TITLE
TP4 Ejercicio 8 - Busqueda

### DISCUSSION THREAD

**Discussion by sanvaltos** [INGCOM-UNRN-P1/cursada-2023#141] [Preguntas y respuestas] [answered]
Hola, no entiendo por qué la declaracion que nos dan en el ejercicio 8 esta declarada como un puntero ¿quiere decir que la funcion 'busqueda' devuelve es un puntero a 'char'?

Prototype: 

```C
char* busqueda(char cadena[], int capacidad_cadena, char busqueda[], int capacidad_busqueda);
```

**Comment by martinvilu** [comment, marked as answer]
De la consigna;
> Implementar una función que retorne el puntero relativo a una cadena segura

La idea es usar aritmetica de punteros

**Reply by martinvilu** [reply]
> ¿quiere decir que la funcion '`busqueda`' devuelve es un puntero a 'char'?

Sí, y relativo a `cadena`
